spec/lib/faraday/zipkin-tracer_spec.rb

```diff
@@ -17,29 +17,29 @@ def on_complete
   end
 
   # Ruby 1.8 didn't support \h
-  HEX_REGEX = RUBY_VERSION >= "1.9.2" ? '\h' : '[0-9a-fA-F]'
+  HEX_REGEX = RUBY_VERSION >= '1.9.2' ? '\h' : '[0-9a-fA-F]'
 
-  let(:response_env) { { :status => 200 } }
-  let(:wrapped_app) { lambda{|env| ResponseObject.new(env, response_env)} }
+  let(:response_env) { { status: 200 } }
+  let(:wrapped_app) { lambda{ |env| ResponseObject.new(env, response_env) } }
```

Comment:
what about using the stabby syntax (from 1.9.2)?

`-> (env) { ResponseObject.new(env, response_env) }`